bokehjs/src/coffee/models/tools/gestures/wheel_pan_tool.coffee

```diff
@@ -6,18 +6,7 @@ import * as p from "../../../core/properties"
 export class WheelPanToolView extends GestureToolView
 
   _scroll: (e) ->
-    # we need a browser-specific multiplier to have similar experiences
-    if navigator.userAgent.toLowerCase().indexOf("firefox") > -1
-      multiplier = 20
-    else
-      multiplier = 1
-
-    if e.originalEvent?.deltaY?
-      delta = -e.originalEvent.deltaY * multiplier
-    else
-      delta = e.bokeh.delta
```

Comment:
OK will test all browsers on OS X an some windows VMs. Can you check several linux combinations?